### PR TITLE
Adds over ONE HUNDRED new clothing options to yogs

### PR DESCRIPTION
**Pull request by fluffe9911**
For some reason after finding a list of codebases and noticing some of them were dead and or dying I decided to go ahead and take some of the more neat unique clothes from there and add it to yogs cause I guess im bloody insane or something anyway clothe sprites have been mostly taken from
https://github.com/UristMcStation/UristMcStation
https://github.com/NTStation/NTstation13
and https://github.com/OracleStation/OracleStation
with a couple of sprites also being taken from hippie, /vg/, the unused folder for tg station and the unused folder for baystation im also sure there are other clothes from other codebases I have missed so be sure to let me know that is all and hope it was worth the wait!

Also most clown sprites were made by SvankHead5000 which is neat

Changelog
🆑
rscadd: Adds over one hundred new clothing options to the game!
/🆑

### CODE REVIEW

File: code/modules/vending/autodrobe.dm [hunk 1]
```diff
@@ -15,35 +15,76 @@
...
+					/obj/item/clothing/head/yogs/dresden = 1, //yogs
+					/obj/item/clothing/under/yogs/doomsday = 1, //yogs
 					/obj/item/clothing/head/that = 1,
 					/obj/item/clothing/under/kilt = 1,
 					/obj/item/clothing/head/beret = 1,
 					/obj/item/clothing/under/yogs/lederhosen = 1, //yogs
 					/obj/item/clothing/head/yogs/folkhat = 1, //yogs
 					/obj/item/clothing/accessory/waistcoat = 1,
-					/obj/item/clothing/glasses/monocle =1,
+					/obj/item/clothing/glasses/monocle = 1,
```
Comment (Altoids1) [inline code/modules/vending/autodrobe.dm]: Don't change ``/tg/`` lines w/o a ``//yogs``, doofus

File: code/modules/vending/autodrobe.dm [hunk 2]
```diff
@@ -15,35 +15,76 @@
...
 					/obj/item/clothing/head/kitty = 1,
 					/obj/item/clothing/under/skirt/black = 1,
+					/obj/item/clothing/under/yogs/zootsuit = 1, //yogs
+					/obj/item/clothing/head/yogs/zoothat = 1, //yogs
+					/obj/item/clothing/under/yogs/hamiltonuniform = 1, //yogs
+					/obj/item/clothing/suit/yogs/hamiltoncoat = 1, //yogs
 					/obj/item/clothing/head/beret = 1,
 					/obj/item/clothing/accessory/waistcoat = 1,
 					/obj/item/clothing/under/suit_jacket = 1,
+					/obj/item/clothing/suit/hooded/sandsuit = 1, //yogs
```
Comment (Altoids1) [inline code/modules/vending/autodrobe.dm]: There are a *lot* of patches of ``//yogs`` lines in this file.

Do these need to be in a particular order? Can you not just put them all together and just have a nice, big ``//yogs start //yogs end`` block? It'll help reduce the shittiness of dealing with any merge conflicts.

An even sexier alternative would be to take these all out and, in ``/obj/machinery/vending/autodrobe/Initialize()``, do, like,
```dm
/obj/machinery/vending/autodrobe/Initialize()
	..()
	products += list(
		obj/fuckingthingA = 1,
		obj/fuckingthingB = 1,
		/* Et cetera, et cetera... */
	)
```
Assuming Initialize for this thing is not already defined by /tg/, which I don't think it is.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 3]
```diff
@@ -177,12 +193,20 @@
...
 	new /obj/item/clothing/under/rank/det(src)
+	new /obj/item/clothing/under/yogs/forensictech(src) //yogs
 	new /obj/item/clothing/suit/det_suit(src)
 	new /obj/item/clothing/head/fedora/det_hat(src)
 	new /obj/item/clothing/gloves/color/black(src)
 	new /obj/item/clothing/under/rank/det/grey(src)
+	new /obj/item/clothing/under/yogs/bluedetective(src) //yogs
+	new /obj/item/clothing/under/yogs/golddetective(src) //yogs
+	new /obj/item/clothing/under/yogs/greydetective(src) //yogs
+	new /obj/item/clothing/under/yogs/blackdetective(src) //yogs
```
Comment (Altoids1) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: Do the clumping thing over here, too.

**Review by Altoids1 — DISMISSED** [review]

File: code/modules/vending/autodrobe.dm (hunk 1 above)
Comment (fluffe9911) [reply code/modules/vending/autodrobe.dm]: oh n o I added a single space! What Horrors!!!!

File: yogstation/code/modules/vending/wardrobes.dm [hunk 4]
```diff
@@ -26,4 +28,181 @@
...
+					/obj/item/clothing/accessory/waistcoat = 2,
+					/obj/item/clothing/suit/apron/purple_bartender = 2,
+					/obj/item/clothing/head/soft/black = 2,
+					/obj/item/clothing/shoes/sneakers/black = 2,
+					/obj/item/reagent_containers/glass/rag = 2,
+					/obj/item/storage/box/beanbag = 1,
+					/obj/item/clothing/suit/armor/vest/alt = 1,
+					/obj/item/circuitboard/machine/dish_drive = 1,
+					/obj/item/clothing/glasses/sunglasses/reagent = 1,
+					/obj/item/storage/belt/bandolier = 1) // yogs - removed pet collar
```
Comment (monster860) [inline yogstation/code/modules/vending/wardrobes.dm]: > having a //yogs in a yogstation folder

File: yogstation/code/modules/vending/clothesmate.dm [hunk 5]
```diff
@@ -1,94 +0,0 @@
-/obj/machinery/vending/clothing
```
Comment (monster860) [inline yogstation/code/modules/vending/clothesmate.dm:1]: Don't delete files just put a comment saying it's mirrored or something

**Review by monster860 — CHANGES_REQUESTED** [review]

File: yogstation/code/modules/vending/clothesmate.dm (hunk 5 above)
Comment (monster860) [reply yogstation/code/modules/vending/clothesmate.dm:1]: oh wait thats a yogs file

Comment (fluffe9911) [reply yogstation/code/modules/vending/clothesmate.dm:1]: monster pls

**Review by monster860 — APPROVED** [review]

**Pull request merged**